ios/RnLdk.swift

```diff
@@ -98,7 +98,7 @@ class MyFilter: Filter {
     override func register_output(output: WatchedOutput) -> Option_C2Tuple_usizeTransactionZZ {
         print("ReactNativeLDK: register_output")
         let scriptPubkeyBytes = output.get_script_pubkey()
-        let outpoint = output.get_outpoint()
+        let outpoint = output.get_outpoint()!
```

Comment:
whats this for?

Reply:
LDKOutPoint is now treated as a nullable type, which affects all points where LDKOutPoint is returned. With some better heuristics in the future, I will hopefully  be able to only make it nullable in some calls' return values and not others.